Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -86,12 +86,6 @@ PoolMultiThreader::PoolMultiThreader()
   }
 
   ThreadIdType defaultThreads = std::max(1u, GetGlobalDefaultNumberOfThreads());
-#if !defined(ITKV4_COMPATIBILITY)
-  if (defaultThreads > 1) // one work unit for only one thread
-  {
-    defaultThreads *= 4;
-  }
-#endif
```

Comment:
I don't know exactly what this code is for, but it is meant for the case that ITKV4_COMPATIBILITY is _not_ defined. So should this code really be removed?